cmd/webhook-manager/app/options/options.go

```diff
@@ -68,6 +69,7 @@ func (c *Config) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&c.WebhookNamespace, "webhook-namespace", "", "The namespace of this webhook")
 	fs.StringVar(&c.WebhookName, "webhook-service-name", "", "The name of this webhook")
 	fs.StringVar(&c.WebhookURL, "webhook-url", "", "The url of this webhook")
+	fs.StringVar(&c.EnabledAdmission, "enabled-admission", "", "enabled admission webhooks")
```

Comment:
we can define `defaultEnabledAdmission` and use here by default

Reply:
done